src/main/java/org/opentripplanner/model/StationElement.java

```diff
@@ -42,6 +44,7 @@ public StationElement(
   /**
    * Name of the station element if provided.
    */
+  @Nullable
```

Comment:
When looking at the code. I think this should be `@Nonnull`, and a default value to be supplied to it instead

Reply:
What should the default value be? Some generic string like "node"?